controller.rst

```diff
@@ -587,6 +587,44 @@ using the ``type`` option of the attribute::
 
     The ``type`` option of ``#[MapRequestPayload]`` was introduced in Symfony 7.1.
 
+Mapping Uploaded File
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can map ``Uploaded Files`` to the controller arguments and add ``constraints``.
+By default the argument resolver will try to match the argument name with the submitted file name.
+
+The constraints defined by you are performed before injecting the uploaded file into the controller argument.
+An ``Unprocessable Content HTTP``` exception is thrown if a constraint violation is detected.
+
+    #[Route('/user/picture', methods: ['PUT'])]
```

Comment:
Please add the use statements of the UploadedFile, the Assert and the MapUploadedFile